civis/tests/mocks.py

```diff
@@ -75,7 +75,7 @@ def create_client_mock_for_container_tests(
         mock_container_run['error'] = 'None'
     c.scripts.post_containers_runs.return_value = mock_container_run_start
     c.scripts.get_containers_runs.return_value = mock_container_run
-    c.scripts.list_containers_runs_outputs.return_value = (run_outputs or [])
+    c.jobs.list_runs_outputs.return_value = (run_outputs or [])
```

Comment:
Related to the comment above -- try mocking both `scripts.list_containers_runs_outputs` and `jobs.list_runs_outputs`.

Reply:
That makes complete sense, thank you!